modules/order/src/Tests/OrderEmptyStoreTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\commerce_order\Tests\OrderEmptyStoreTestBase,
+ * Contains \Drupal\commerce_order\Tests\OrderEmptyStoreTest.
+ */
+namespace Drupal\commerce_order\Tests;
+
+use Drupal\commerce\Tests\CommerceTestBase;
+use Drupal\Core\Url;
+use Drupal\field\Tests\EntityReference\EntityReferenceTestTrait;
+use Drupal\Core\Entity\EntityInterface;
+/**
+ * Defines base class for Empty store test cases.
+ */
+abstract class OrderEmptyStoreTestBase extends CommerceTestBase {
```

Comment:
You should not be creating a new test base class, use OrderTestBase instead, add the rest in OrderEmptyStoreTest